account_analytic_sequence/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Analytic Sequence',
+    'summary': """
+        Restore the analytic account sequence""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://www.acsone.eu',
```

Comment:
This should point to OCA repository as per https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11